lib/src/gauge_group.rs

```diff
@@ -3,46 +3,76 @@ use std::{
     sync::{Arc, Weak},
 };
 
+use crate::pipeline::aggregation::Aggregation;
+use crate::pipeline::aggregator::DimensionedMeasurementsMap;
 use crate::{
     gauge::{self, StatisticSetGauge},
-    pipeline::aggregation::statistic_set::StatisticSet,
+    pipeline::aggregator::DimensionPosition,
     types::Name,
 };
 
 #[derive(Default)]
 pub struct GaugeGroup {
-    gauges: HashMap<Name, Weak<StatisticSetGauge>>,
+    pub(crate) dimensioned_gauges:
```

Comment:
Why is this pub(crate)?

Reply:
I think just a leftover from some earlier code I had, I'll remove.